client/extensions/woocommerce/components/query-locations/index.js

```diff
@@ -34,5 +35,5 @@ class QueryLocations extends Component {
 
 export default connect(
 	null,
-	{ fetchLocations }
+	dispatch => bindActionCreators( { fetchLocations }, dispatch )
```

Comment:
This change looks good – just curious, does this mean the address form upon store setup is currently broken in `master`?

Reply:
Yes. This is too far on my past, but I recall that in Calypso it didn't really matter because there was always any other outer component that fetched the locations, so having a few placebo `<QueryLocations>` didn't break anything :)